conf/keycloak/builtin-users-spi/src/main/java/edu/harvard/iq/keycloak/auth/spi/services/PasswordEncryption.java

```diff
@@ -0,0 +1,82 @@
+package edu.harvard.iq.keycloak.auth.spi.services;
```

Comment:
Can you add notes in the code that this and the original class in edu.harvard.iq.dataverse.authorization.providers.builtin have to stay in sync? (This class has been pretty stable but we had plans to switch from BCrypt_10 at one point and that would be needed in both places now.)